tests/unit/resolvers/classic/basic-test.js

```diff
@@ -19,31 +22,37 @@ function setupResolver(options) {
 
 function resetRegistry() {
   requirejs.clear();
-  Ember.merge(requirejs.entries, originalRegistryEntries);
+  merge(requirejs.entries, originalRegistryEntries);
 }
 
 module('ember-resolver/resolvers/classic', {
+  before() {
+    this.sandbox = sinon.sandbox.create();
+  },
+
   beforeEach() {
-    originalRegistryEntries = Ember.merge({}, requirejs.entries);
-    originalEmberDeprecate = Ember.deprecate;
+    originalRegistryEntries = merge({}, requirejs.entries);
+    originalEmberDeprecate = deprecate;
     originalConsoleInfo = console ? console.info : null;
 
     setupResolver();
   },
 
   afterEach() {
     resetRegistry();
-    Ember.deprecate = originalEmberDeprecate;
+    deprecate = originalEmberDeprecate;
     if (originalConsoleInfo) {
       console.info = originalConsoleInfo;
     }
+
+    this.sandbox.restore();
   }
 });
 
 test("can access at deprecated 'resolver' module name", function(assert){
   assert.expect(2);
 
-  Ember.deprecate = function(message, test) {
+  deprecate = function(message, test) {
```

Comment:
Hmm, in general, I'm not sure this works (if it does its essentially a bug in babel-plugin-ember-modules-api-polyfill).  Instead we should use [ember-qunit-assert-helpers](https://github.com/workmanw/ember-qunit-assert-helpers) which provides an `assert.expectDeprecation` system that is pretty handy...